release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1077,6 +1077,13 @@ With this release, control plane machine sets are supported for clusters that ru
 
 For more information, see xref:../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-getting-started[Getting started with the Control Plane Machine Set Operator].
 
+[NOTE]
+====
+For clusters that have root volume availability zones and are running on {rh-openstack} that you upgrade to {product-version} and have root volume availability zones, you must converge control plane machines onto one server group before you can enable control plane machine sets. To make the required change, follow the instructions in the related link:https://access.redhat.com/solutions/7013893[knowledge base article].
+
+For clusters that have compute zones configured with at least one zone and are running on {rh-openstack}, which is upgradable to version {product-version}, root volumes must now also be configured with at least one zone. If this configuration change does not occur, a control plane machine set cannot be generated for your cluster. To make the required change, follow the instructions in the related link:https://access.redhat.com/solutions/7024383[knowledge base article].
```

Comment:
Since these are two different kbase article links, it might be helpful to users to list the titles of the articles, instead of just "knowledge base article" as the link text.

See https://redhat-documentation.github.io/supplementary-style-guide/#rh-kb-links for more info on linking to kbase articles

Reply:
```suggestion
For clusters that have compute zones configured with at least one zone and are running on {rh-openstack}, which is upgradable to version {product-version}, root volumes must now also be configured with at least one zone. If this configuration change does not occur, a control plane machine set cannot be generated for your cluster. To make the required change, follow the instructions in the related link:https://access.redhat.com/solutions/7024383[OpenShift on OpenStack with compute Availability Zones: Missing rootVolume availability zone].
```